Makefile

```diff
@@ -129,34 +129,50 @@ lint-go:
 
 # Docker
 
-build-push-docker:
-	@$(MAKE) build-docker registry=$(REGISTRY) version=$(VERSION)
-	@$(MAKE) push-ci-docker registry=$(REGISTRY) version=$(VERSION)
-	@$(MAKE) push-core-docker registry=$(REGISTRY) version=$(VERSION)
-	@$(MAKE) push-serving-docker registry=$(REGISTRY) version=$(VERSION)
-
-build-docker: build-ci-docker build-core-docker build-serving-docker
+build-docker: build-ci-docker build-feature-server-aws-docker build-feature-transformation-server-docker build-feature-server-java-docker
 
 push-ci-docker:
 	docker push $(REGISTRY)/feast-ci:$(VERSION)
 
+# TODO(adchia): consider removing. This doesn't run successfully right now
 build-ci-docker:
 	docker build -t $(REGISTRY)/feast-ci:$(VERSION) -f infra/docker/ci/Dockerfile .
 
-build-local-test-docker:
-	docker build -t feast:local -f infra/docker/tests/Dockerfile .
-
-push-core-docker:
-	docker push $(REGISTRY)/feast-core:$(VERSION)
+# Note, we use underscores instead of periods in the version name since ECR doesn't support periods. We automatically
+# pull from Docker Hub and push to ECR.
+push-feature-server-aws-docker:
+	ECR_VERSION=`echo "$(VERSION)" | sed 's/[.]/_/g'`
+	docker push $(REGISTRY)/feature-server-aws:$(ECR_VERSION)
+
+build-feature-server-aws-docker:
+	ECR_VERSION=`echo "$(VERSION)" | sed 's/[.]/_/g'`
+	docker build --build-arg VERSION=${ECR_VERSION} \
+		-t $(REGISTRY)/feature-server-aws:${ECR_VERSION} \
+		-t gcr.io/kf-feast/feature-server-aws:${ECR_VERSION} \
+		-t gcr.io/kf-feast/feature-server-aws:$(GITHUB_SHA) \
+		-f sdk/python/feast/infra/feature_servers/aws_lambda/Dockerfile .
+
+push-feature-transformation-server-docker:
+	docker push $(REGISTRY)/feature-transformation-server:$(VERSION)
+    docker push gcr.io/kf-feast/feature-transformation-server:$(VERSION)
+
+build-feature-transformation-server-docker:
+	docker build --build-arg VERSION=$(VERSION) \
+		-t $(REGISTRY)/feature-transformation-server:$(VERSION) \
+		-t gcr.io/kf-feast/feature-transformation-server:$(VERSION) \
+		-t gcr.io/kf-feast/feature-transformation-server:$(GITHUB_SHA) \
+		-f sdk/python/feast/infra/transformation_servers/Dockerfile .
 
-push-serving-docker:
-	docker push $(REGISTRY)/feast-serving:$(VERSION)
-
-build-core-docker:
-	docker build --build-arg VERSION=$(VERSION) -t $(REGISTRY)/feast-core:$(VERSION) -f infra/docker/core/Dockerfile .
+push-feature-server-java-docker:
+	docker push $(REGISTRY)/feature-server-java:$(VERSION)
+    docker push gcr.io/kf-feast/feature-server-java:$(VERSION)
```

Comment:
A little while ago we migrated to docker hub as a default container registry? Are we moving back to gcr.io?